docs/tutorials/configure/build-a-mock-robot.md

```diff
@@ -20,50 +20,67 @@ date: "2022-10-11"
 cost: "0"
 ---
 
-This tutorial will show you how to build a mock robot using just your personal laptop so you can:
+In this tutorial you will build a mock robot to learn how to configure robots with Viam.
+You do not need any hardware to do this tutorial.
 
-- Learn how to [configure](../../../manage/configuration/) robots with Viam.
-- Try using [Viam](/viam/) without any robotic hardware.
+Follow this tutorial to set up and control a robot with a `fake` [arm](/components/arm/fake/), [board](/components/board/), and [motor](/components/motor/), and an additional mock {{< glossary_tooltip term_id="part" text="sub-part" >}} with a [motor](/components/motor/).
+These `fake` components interact with Viam like real hardware but do not physically exist.
 
-Most Viam [components](../../../components/) have a _fake_ model that you can use for testing.
-These fake components interact with Viam like real hardware but do not actually exist.
+### What is a sub-part?
```

Comment:
Should we wait with explaining a sub-part till we do that part? Also the fake component section - that explanation might work best when we are picking the model

Reply:
I'm fine with that, will move and comment here when done

Reply:
Moved! And moved out the fake component info. Would have added that to relevant section as well but it didn't add any new info, really, besides what we already had in there, so just deleting